hw/core-v-mini-mcu/core_v_mini_mcu.sv.tpl

```diff
@@ -53,6 +53,8 @@ ${pad.core_v_mini_mcu_interface}
     output obi_req_t  ext_dma_addr_ch0_req_o,
     input  obi_resp_t ext_dma_addr_ch0_resp_i,
 
+    input logic [core_v_mini_mcu_pkg::DMA_CH_NUM-1:0] dma_stop_i,
```

Comment:
`ext_dma_stop_i`

Reply:
also, @TommiTerza , add a comment saying this signal stops the DMA from from a controller outside.. also, why is this here? shouldn't this belong to the next PR? however does not matter, just comment it

Reply:
You are right, let's say it's a bit of a spoiler :)